packages/client/components/TeamPrompt/TeamPromptResponseCard.tsx

```diff
@@ -30,23 +30,26 @@ const fiveColumnResponseMediaQuery = `@media screen and (min-width: ${ResponsesG
 
 const ResponseWrapper = styled('div')<{
   status: TransitionStatus
-}>(({status}) => ({
+  isSingleColumn: boolean
+}>(({status, isSingleColumn}) => ({
   opacity: status === TransitionStatus.MOUNTED || status === TransitionStatus.EXITING ? 0 : 1,
   transition: `box-shadow 100ms ${BezierCurve.DECELERATE}, opacity 300ms ${BezierCurve.DECELERATE}`,
   display: 'flex',
   flexDirection: 'column',
   width: '100%',
+  maxWidth: isSingleColumn ? '850px' : undefined,
```

Comment:
850px is somewhat arbitrary - it's a little more than the previous single-column max width

Reply:
I had landed on [600px on the mockups ](https://www.figma.com/file/mR6vYT8fUDRv9tBGb8OPCL/Standups-iterations?type=design&node-id=1572-25292&mode=design&t=DGUyiQtSLxHm4eLd-4) because I felt that wider than that the text readability suffers, since long text lines are harder on the eye.

Do you think that's too narrow?

Reply:
Fair enough! Updated this to 600px instead.